src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1109,12 +1109,23 @@ internal struct Slot
         }
     }
 
+    /// <summary>A placeholder class for common padding constants and eventually routines.</summary>
+    internal static class PaddedHelper
```

Comment:
We now have two places where we define the cache line size (at the other place, we use just the 128 bytes for all platforms). I wonder if we should unify it and put it e.g. to System.Environment.
@jkotas, would the System.Environment be the right place?